lib/assets/stylesheets/modules/_technical-documentation.scss

```diff
@@ -123,6 +123,12 @@
     overflow: auto;
     position: relative;
     border: 1px solid $code-02;
+
+    &:focus {
+      padding: 13px;
```

Comment:
I'd be tempted to add a comment explaining where this 'magic number' has come from / what it's doing.

Alternatively, you could 'show your working' by doing something like this:

```scss
  pre {
    $padding: 15px;
    $border-width: 1px;

    background: $code-00;
    padding: $padding;
    overflow: auto;
    position: relative;
    border: $border-width solid $code-02;

    &:focus {
      padding: $padding - ($govuk-focus-width - $border-width);
      border: $govuk-focus-width solid $govuk-focus-text-colour;
      outline: $govuk-focus-width solid $govuk-focus-colour;
    }
}
```

That also means everything's more likely to stay in sync if e.g. someone came in and changed the padding or border width.

Reply:
Yep, good shout. Updated in 6467c8abc13497482da62f12ce4b16c6a3a1460e